lib/client/client.go

```diff
@@ -137,7 +137,7 @@ func (c *Client) LoadTransaction(id string, queries ...Q) (transaction Transacti
 	return
 }
 
-func (c *Client) LoadTransactionHistory(id string, queries ...Q) (transaction Transaction, err error) {
+func (c *Client) LoadTransactionHistory(id string, queries ...Q) (transaction TransactionHistory, err error) {
```

Comment:
@kfangw How about modifying the variable `transaction` to `txh` or `transactionHistory` also?